### ISSUE TITLE
After update to Seq 2.4.2 integration is broken

### ISSUE THREAD

**Issue by sergiorykov** [bytenik/Seq.App.Slack#6]
Full stack trace:

```
System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Seq.Slack.SlackReactor.SubstitutePlaceholders(String messageTemplateToUse, Event`1 evt)
   at Seq.Slack.SlackReactor.GenerateMessageText(Event`1 evt)
   at Seq.Slack.SlackReactor.On(Event`1 evt)
   at Seq.Server.Features.Runner.AppHost.SendLogEventData(ISubscribeTo`1 reactor, DateTime utcTimestamp, Int64 arrivalOrder, UInt32 hash, String document)
   at Seq.Server.Features.Runner.AppHost.Send(StorageEvent storageEvent)
```

**Comment by sergiorykov** [comment]
In manual mode it works. Fails only when use filtering signal.

**Comment by bytenik** [comment]
Fixed in 0.1.9 of the integration, which I just published to NuGet.
https://www.nuget.org/packages/Seq.App.Slack/0.1.9

**Comment by sergiorykov** [comment]
Thanks!
